recipes/spirv-reflect/all/conanfile.py

```diff
@@ -0,0 +1,54 @@
+import os
+
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get
+
+required_conan_version = ">=2.0"
+
+
+class SpirvReflectConan(ConanFile):
+    name = "spirv-reflect"
+    homepage = "https://github.com/KhronosGroup/SPIRV-Reflect"
+    description = ("SPIRV-Reflect is a lightweight library that provides a C/C++ reflection API "
+                   "for SPIR-V shader bytecode in Vulkan applications.")
+    license = "Apache-2.0"
+    topics = ("spirv", "spirv-v", "vulkan", "opengl", "opencl", "khronos")
+    url = "https://github.com/conan-io/conan-center-index"
+    package_type = "static-library"
```

Comment:
The linter is warning against a mising `fPIC` option, if you could add it back then the rest looks good to me for this one, thanks a lot!